src/org/opendatakit/briefcase/ui/BriefcaseCLI.java

```diff
@@ -55,6 +55,7 @@
   private static final String ODK_PASSWORD = "odk_password";
   private static final String ODK_USERNAME = "odk_username";
   private static final String OVERWRITE_CSV_EXPORT = "overwrite_csv_export";
+  private static final String PULL_BEFORE_EXPORT = "pull_before";
```

Comment:
Don't create the param here. This is the legacy Launcher and we want users to switch to the new Launcher. Let's use the new features to get them to the new Launcher.

Reply:
ok